docs/pages/visual-test/BannerOverlay-dark.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import { type Node as ReactNode } from 'react';
+import { BannerOverlay, ColorSchemeProvider, FixedZIndex, Image, Link, Text } from 'gestalt';
+
+export default function Example(): ReactNode {
+  return (
+    <ColorSchemeProvider colorScheme="dark">
+      <BannerOverlay
```

Comment:
Can you reference the light snapshot here, since they are the same? This is a new pattern. Take a look at SegmentedControl. That way we only have to update one

Reply:
Done!